### PR TITLE
feat: Hero 'Media(Blend)'

### LINKED ISSUE #79: Styling of WebComponent HeroBanner
Goal: The HeroBanner styling on article/blog pages matches the design given on https://www.figma.com/file/oSetT4LbatRmXlcB2A7V8Z/Content-Hubs-2024?type=design&node-id=269-30889&mode=design&t=D0jJGSxZlJFSCg4b-0

Reference page (AEM EDS): https://main--hlx-test--urfuwo.hlx.live/blog/2023/12/being-human-in-the-age-of-ai

List of concerns:
* Text box padding and margin (spacing to the page border)
* Bounding box of the text (within the element)
* Technical integration of the background image (see EDS block hero.js code)

### PR DESCRIPTION
**Pull request by sdmcraft**
Fix #79 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/draft/satyam/idc-marketscape-sap-a-leader-cloud-enabled-sourcing-applications
- After: https://hero--hlx-test--urfuwo.hlx.live/draft/satyam/idc-marketscape-sap-a-leader-cloud-enabled-sourcing-applications

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation | Library Link
------------- | -------------|----------------
social | [doc-link](https://sap.sharepoint.com/sites/207899/_layouts/15/doc.aspx?sourcedoc={5c2d8d7b-f6d5-4356-b71c-3730ad5886db}&action=edit) | [Library Link](https://social--hlx-test--urfuwo.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/tools/sidekick/blocks/social&index=1)

- [ ] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
 For e.g. hero (light)  | [doc-link](https://sap.sharepoint.com/:w:/r/sites/207899/_layouts/15/Doc.aspx?sourcedoc=%7B0B62F23A-5C8F-4C2D-BA2E-C444D3631B51%7D&file=hero.docx&action=default&mobileredirect=true)

### CODE REVIEW

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -1,31 +1,177 @@
-main .hero-container > div {
-  max-width: unset;
+/* stylelint-disable selector-class-pattern */
```
Comment (sdmcraft) [inline blocks/hero/hero.css:1]: Adding these styles temporarily. Ideally these styles should come bundled with the `udex-hero-banner` web-component. 
/cc @ok

**Comment by sdmcraft** [comment]
Spillover items covered in https://github.com/urfuwo/hlx-test/issues/135

**Review by karlpauls — APPROVED** [review]
Nice!

**Comment by saurabh-khare** [comment]
@sdmcraft Looks nice to me. Just curious on how have we evaluated container breakpoints? Are they part of component specs. 
Also I think the image is small compared to text just before S breakpoint
Screenshot 2024-02-16 at 13 09 18

**Review by saurabh-khare — APPROVED** [review]

**Comment by sdmcraft** [comment]
> Just curious on how have we evaluated container breakpoints? Are they part of component specs.

Yes. I copied over the styles from the UDex spec for `hero-banner` under the impression that this is temporary but looks like we indeed have to specify the styles outside of the web-component. Given this, I'll clean need to up the styles.

**Comment by karlpauls** [comment]
One thing that right now is not optimal too is the fonts handling - the property for the font-family seems not to be referencing a font with a swap and fallback. As the blend variant is making the text the LCP I think this might be costing us some points.

**Comment by karlpauls** [comment]
@sdmcraft, I think now you'd have to adapt the other variables that reference the `72 Brand` font as well (not sure if they are used but anyways). Furthermore, not sure if it really is an issue but I think we'll need to setup a fallback font that is sized to match the original (https://www.aem.live/developer/font-fallback) - at least the last time, I had an article where the text was the LCP (which then might give CLS).

**Comment by benpeter** [comment]
@karlpauls @sdmcraft good to merge and do the rest in #136 ?

**Pull request merged**